docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/MapKeyToVector.cs

```diff
@@ -67,8 +67,8 @@ private class DataPoint
 
         private class TransformedData : DataPoint
         {
-            public uint[] TimeframeVector { get; set; }
-            public uint[] CategoryVector { get; set; }
+            public float[] TimeframeVector { get; set; }
```

Comment:
the other option is leave it as uint[] but add another transform in the pipeline that converts these column types from float to uint but this is not currently possibly because we don't have float to uint converter in Conversions.cs but of course that can be added and we can use C#'s convention for conversion to uint by casting.

CC: @TomFinley

Reply:
I think changing the type is appropriate -- this is a key-to-vector sample, the output type of key-to-vector is a float vector, so, this change seems appropriate at least to me.

Reply:
Thanks @TomFinley